### PR TITLE
microsoft-defender: new Cask

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: Casks/microsoft-defender.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  auto_updates true
+  depends_on cask: "microsoft-auto-update"
+  depends_on macos: ">= :big_sur"
+
+  pkg "Microsoft Defender.pkg",
+      choices: [
+        {
+          "choiceIdentifier" => "com.microsoft.autoupdate", # Office16_all_autoupdate.pkg
```
Comment (SMillerDev) [inline Casks/microsoft-defender.rb:22]: What about office?

Comment (miccal) [reply Casks/microsoft-defender.rb:22]: Removing the comment; it is not needed here.

**Review by SMillerDev — DISMISSED** [review]

File: Casks/microsoft-defender.rb (hunk 1 above)
Comment (miccal) [reply Casks/microsoft-defender.rb]: Good catch @SMillerDev, thank you -- I copied this from the `microsoft-office` Cask but I have now realised that for this Cask it has a different `choiceIdentifier.`

Comment (miccal) [reply Casks/microsoft-defender.rb]:
```
<dict>
	<key>childItems</key>
	<array/>
	<key>choiceIdentifier</key>
	<string>com.microsoft.package.Microsoft_AutoUpdate.app</string>
	<key>choiceIsEnabled</key>
	<true/>
	<key>choiceIsSelected</key>
	<integer>1</integer>
	<key>choiceIsVisible</key>
	<false/>
	<key>choiceSizeInKilobytes</key>
	<integer>15045</integer>
	<key>pathsOfActivePackagesInChoice</key>
	<array>
		<string>file://localhost/Users/00084858/Downloads/Microsoft%20Defender.pkg#MAU.pkg</string>
	</array>
</dict>
```

**Review by p-linnane — APPROVED** [review]

**Pull request merged**